### DISCUSSION TITLE
blog/duty-of-sdet

### DISCUSSION THREAD

**Discussion by giscus** [CarlJi/website#4] [General]
# blog/duty-of-sdet

网上有人会把测开定位成为 测试工具开发，主要是开发自动化测试工具或平台，用以帮助手动验收的同学提升效率。存在即合理，确实有一些团队或组织是这样建设的。但作为行业从业者，我们也应该认识到，这样是不全面的，有误导之嫌。

https://carlji.netlify.app/blog/duty-of-sdet

**Comment by CarlJi** [comment]
测试comment
